_projects/2022/100481925/100481925.Rmd

```diff
@@ -0,0 +1,1481 @@
+categories: "2022"
+author: Carolina Cornejo
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+## Introduction
+On September 25, 2022, the [_Visual Capitalist_](https://www.visualcapitalist.com/), a digital media focused on generating data-centric visual content, published the article _How Do Americans Spend Their Money, By Generation?_ The author of the visualization is Preethi Lodha, and the author of the article is Carmen Ang. 
+
+This report aims to discuss Preethi Lodha’s graph in the light of data visualization theory. Thus, some questions that will guide this narrative are: What information does it intend to show? Does it achieve it? How and through what encodings and channels? What are its strengths and points of improvement? Likewise, improvements and alternative visualizations are proposed, which are also not without its own room for improvement. 
+
+## Background 
+The article collected data from the 2021 Consumer Expenditure Surveys (CES) from the [U.S. Bureau of Labor Statistics](https://www.bls.gov/cex/tables/calendar-year/mean-item-share-average-standard-error/reference-person-age-generation-2021.pdf). Specifically, the data used for the visualization was that published in PDF format in _Table 2602. Generation of reference person: Annual expenditure means, shares, standard errors, and coefficients of variation_.
+
+The source of the article and the original chart can be accessed [here](https://www.visualcapitalist.com/cp/how-americans-spend-their-money-2022/). This report was written in December 2022.
+
+## What is it for? Representing how Americans distributed their spending in 2021
+First, the context and intentionality must be understood. The graphics must be understandable by themselves, and also in context.
+
+As for the article on the _Visual Capitalist_ website, it has as newsworthiness -in Spanish journalist jargon we call it "la pepa"- to show the distribution of Americans' spending in 2021. Although the title of the article does not specify that the information is from the previous year -it was written and published in 2022, but the information is from 2021- this type of generalization or omissions are common in the titles and/or in the headlines of journalistic articles: they are licenses that journalists give themselves to attract the reader's attention. The content of the article clearly specifies the source of information—the PDF—and that it refers to 2021. So it's not a time series, it's just a picture of that moment.
+
+<div>
+  <center><img src="https://github.com/carolinacornejocastellano/bump-chart/blob/main/original.png?raw=true" width="100%">
+    <figcaption>
+    <span class="caption">Screenshot of the public Tableau dashboard 'How Americans spend their money', by Preethi Lodha.</span>
+    <i class="photo-credit">Published in: https://www.visualcapitalist.com/cp/how-americans-spend-their-money-2022/</i>
+  </figcaption>
+  </p>
+</div>
```

Comment:
I would prefer if you let pandoc generate the HTML for these figures. So please use Markdown syntax for all the images you add with divs. For instance, this one would be just:

```markdown
![Screenshot of the public Tableau dashboard 'How Americans spend their money', by Preethi Lodha.  Published in: https://www.visualcapitalist.com/cp/how-americans-spend-their-money-2022/](https://github.com/carolinacornejocastellano/bump-chart/blob/main/original.png?raw=true){.external width="100%"}
```

Note the `.external` tag. This adds a special border intended for external images. So do not add it to your own images.

Also, I would prefer if posts do not depend on external resources as much as possible. So please download all these images and place them into the `_projects/2022/100481925/images` folder I've created. Then you can reference them with a path relative to the Rmd, so the code above would be just

```markdown
![Screenshot of the public Tableau dashboard 'How Americans spend their money', by Preethi Lodha.  Published in: https://www.visualcapitalist.com/cp/how-americans-spend-their-money-2022/](images/original.png){.external width="100%"}
```